### ISSUE TITLE
Suggestion: slightly change colour scheme so it meets accessibility recommendations

### ISSUE THREAD

**Issue by HelmetOwl** [teachingtechYT/teachingtechYT.github.io#421]
Some of the text on the site does not contrast well against its background on the site, which makes it difficult to read for users with some forms of visual impairment.

For example, the red text on red background here:
<img width="1382" alt="Screenshot 2022-07-05 at 18 07 38" src="https://user-images.githubusercontent.com/86771201/177280927-f79a1472-02b8-46cf-b160-ba0fd2e84777.png">
The required contrast ratio is 4.5 but it only has a contrast ration of 2.81. The green text on red background is even worse, with a contrast ratio of 1.54. The white text on green background on the other parts of the page fails too.

There are colour accessibility calculators online. My favourite is [accessible-colors.com](https://accessible-colors.com/), because it suggests alternative colours that bring it in line with accessibility requirements. Alternatives are: [webaim.org](https://webaim.org/resources/contrastchecker/) and [accessibleweb.com](https://accessibleweb.com/color-contrast-checker/), which both have other web accessibility tools too.

There's a guide on accessible colour and contrast from the South Australian Government here: [Colour and contrast](https://www.accessibility.sa.gov.au/your-role/visual-design/colour-and-contrast).
